### DISCUSSION TITLE
Am I misunderstanding how the activation constraints work somehow? The delay property does not seem to be working as I thought it would.

### DISCUSSION THREAD

**Discussion by zachkaigler** [clauderic/dnd-kit#1470] [Q&A]
Pull up this [super simple sandbox](https://codesandbox.io/p/sandbox/keen-lehmann-2rtxgm) and open the dev tools console.

Click on the blue box and drag. Note that in the console you see messages alerting drag start and drag end. Refresh the preview and click on the blue box without dragging. Note that you'll see the same messages in the console.

Open up `App.tsx` and check out the activation constraints that I'm passing into `useSensor`:

```
  const pointerSensor = useSensor(PointerSensor, {
    activationConstraint: {
      delay: 5000,
      tolerance: 0,
    },
  });
```

My understanding is that this should delay the drag event emission (and subsequently the draggability of the element itself) for five seconds, but instead you can drag immediately. My goal here is to implement a small delay before the drag event fires, so that one-off clicks don't call the `onDragStart` and `onDragEnd` methods exposed by `useDndMonitor`.

Am I fundamentally misunderstanding how these constraints work somehow?

**Comment by clauderic** [comment]
You're using `useDraggable` outside of `<DndContext>` make sure it is used within a component nested within `<DndContext>`

**Reply by zachkaigler** [reply]
`DnDContext` is wrapping `App` from the index.

**Reply by clauderic** [reply]
Yes but you are not passing the sensors to the one that wraps useDraggable
